### PR TITLE
cumulative fixes

### PR DESCRIPTION
**Pull request by axelrod-blox**
IO1-2976-remove 1 validator IO1-3980-remove 1 validator cta IO1-3983 loading state text

### CODE REVIEW

File: src/app/common/stores/applications/SsvWeb/Validator.store.ts [hunk 1]
```diff
@@ -161,19 +161,20 @@ class ValidatorStore extends BaseStore {
...
    */
-  async removeValidator(validator: any): Promise<boolean> {
+  async removeValidator(publicKey: string, operatorIds: number[]): Promise<boolean> {
+    // console.log(validator);
     const clusterStore: ClusterStore = this.getStore('Cluster');
     const notificationsStore: NotificationsStore = this.getStore('Notifications');
     const contract = getContractByName(EContractName.SETTER);
     store.dispatch(setIsLoading(true));
     const myAccountStore: MyAccountStore = this.getStore('MyAccount');
     // @ts-ignore
```
Comment (IlyaVi) [inline src/app/common/stores/applications/SsvWeb/Validator.store.ts]: remove ts-ignore

**Review by IlyaVi — APPROVED** [review]

File: src/app/common/config/translations.ts [hunk 2]
```diff
@@ -48,14 +48,14 @@ const translations = {
           texts: ['Removing your validator will cause your operators to stop managing it in your behalf, which will result in its inactivation (penalties on the Beacon Chain).', 'Please note that this action only applies to its removal from our network and does not exit your validator from the Beacon Chain.'],
           warningMessage: 'To avoid slashing, it is advised to wait at least 2 epochs prior to running the validator on an alternative service.',
           checkBoxes: ['I understand that my validator will be removed from the network and it will stop attesting on the beacon chain'],
-          buttonText: (count: number) => `Remove ${count} validators`,
+          buttonText: (count: number, isLoading: boolean) => `${ isLoading ? 'Removing' : 'Remove'} ${count > 1 ? `${count} validators` : 'validator' }`,
```
Comment (alexstolr) [inline src/app/common/config/translations.ts:51]: you can dynamically create this string by creating a function that gets a string template and variadic params and just returns the entire text. then you can cal it here and in line 58

File: src/app/components/applications/SSV/MyAccount/components/Validator/RemoveFlow/RemoveValidator/RemoveValidator.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ const RemoveValidator = () => {
 
   const removeValidator = async () => {
     if (validator.public_key) {
-      const response = await validatorStore.removeValidator(validator);
+      const response = await validatorStore.removeValidator(validator.public_key, validator.operators);
```
Comment (alexstolr) [inline src/app/components/applications/SSV/MyAccount/components/Validator/RemoveFlow/RemoveValidator/RemoveValidator.tsx:36]: @axelrod-blox  Should the validator.operators be sorted?

**Review by alexstolr — COMMENTED** [review]
already merged but some notes...

**Pull request merged**